collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -504,7 +504,7 @@ public boolean accept(File pathname) {
 					}
 				} else {
 					Log.e(t, "unrecognized URI");
-					this.createErrorDialog("Unrecognized URI: " + uri, EXIT);
+					this.createErrorDialog(getString(R.string.unrecognized_uri) + uri, EXIT);
```

Comment:
Instead of hardcoding the order of the text and URI, the URI should be a parameter so that RTL languages can change the order.